jira7/jira_endpoint.bal

```diff
@@ -321,87 +312,89 @@ public remote function Client.updateProject(string projectIdOrKey, ProjectReques
     //Evaluate http response for connection and server errors
     var jsonResponseOut = getValidatedResponse(httpResponseOut);
 
-    if (jsonResponseOut is JiraConnectorError) {
```

Comment:
Again, where we are returning error without changing anything, we can consider using check.